hack/test/manifests/aws-cluster.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         type: aws
       controlplane:
         count: 3
+        k8sversion: "1.16.1"
```

Comment:
Something for later, but maybe we should align this with what we do in talos and just call it `version` since its already namespaces under `controlplane`